### PR TITLE
TASK: Refactor absolute namespaces in  Flow / Command classes

### PR DESCRIPTION
**Pull request by daniellienert**
Additionally refactor legacy array syntax.

### CODE REVIEW

**Comment by albe** [comment]
The one `Resource` import could be also named `PersistentResource`, but since it's only used in the `@var` doc comment I see this not so much as issue. Looks good :)

File: TYPO3.Flow/Classes/TYPO3/Flow/Command/ResourceCommandController.php [hunk 1]
```diff
@@ -189,7 +191,7 @@ public function cleanCommand()
                             }
                         }
                     }
-                    $this->outputLine('Removed %s resource object(s) from the database.', array(count($brokenResources)));
+                    $this->outputLine('Removed %s resource object(s) from the database.', [count($brokenResources)]);
                 break;
```
Comment (kitsunet) [inline TYPO3.Flow/Classes/TYPO3/Flow/Command/ResourceCommandController.php]: Why remove all this?

Comment (daniellienert) [inline TYPO3.Flow/Classes/TYPO3/Flow/Command/ResourceCommandController.php]: Sorry Christian - what do you mean?

Comment (kitsunet) [inline TYPO3.Flow/Classes/TYPO3/Flow/Command/ResourceCommandController.php]: Uhm, I think it was a display error in the diff view, now iwth the comments it looks normal so nevermind. It looked like the rest of the cases was deleted as well....

**Pull request merged**